pallet_assets.rs

```diff
@@ -0,0 +1,169 @@
+
+//! Autogenerated weights for pallet_assets
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
+//! DATE: 2021-04-21, STEPS: `[20, ]`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("statemint-dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/statemint
+// benchmark
+// --chain
+// statemint-dev
+// --execution
+// wasm
+// --wasm-execution
+// compiled
+// --pallet
+// pallet_assets
+// --extrinsic
+// *
+// --steps
+// 20
+// --repeat
+// 10
+// --raw
+// --output
+// ./
+
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for pallet_assets.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pallet_assets::WeightInfo for WeightInfo<T> {
+	fn create() -> Weight {
+		(68_000_000 as Weight)
```

Comment:
these numbers are now from your local machine right?

Reply:
yup, I put it in the readme and #37 that we need to run it on a proper machine when we are ready